src/modules/tooltip/Utils.js

```diff
@@ -297,8 +297,11 @@ export default class Utils {
     markersWraps.sort((a, b) => {
       return Number(b.getAttribute('data:realIndex')) <
```

Comment:
As we already use a code block, it might be sensible to extract the `getAttribute` out of the check to make it more concise.
```javascript
markersWraps.sort((a, b) => {
  var indexA = Number(a.getAttribute('data:realIndex'))
  var indexB = Number(b.getAttribute('data:realIndex'))
  return indexB < indexA
       ? 1
       : indexB > indexA
       ? -1
       : 0
})
```
If we do not do this, using the short style would also improve legibility:
```javascript
markersWraps.sort((a, b) =>
    Number(b.getAttribute('data:realIndex')) < Number(a.getAttribute('data:realIndex'))
  ? 1
  : Number(b.getAttribute('data:realIndex')) > Number(a.getAttribute('data:realIndex'))
  ? -1
  : 0
)
```